spec/controllers/admin/booths_controller_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'spec_helper'
+
+describe Admin::BoothsController do
+
+  let(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let(:booth) { create(:booth, conference: conference) }
+  let(:admin) { create(:admin) }
+
+  context 'not logged in user' do
+
+    describe 'GET index' do
+      it 'does not render admin/booths#index' do
+        get :index, conference_id: conference.short_title
+        expect(response).to redirect_to(user_session_path)
+      end
+    end
+
+    # describe 'GET show' do
+    #   it 'does not render admin/booths#show' do
+    #     get :show, conference_id: conference.short_title
```

Comment:
There is an obvious error with this, check the parameters on your server logs to find out what's missing.